components/activecampaign/actions/create-or-update-contact/create-or-update-contact.mjs

```diff
@@ -46,6 +46,11 @@ export default {
       description: "(Deprecated) Please use the DELETE endpoint.",
       optional: true,
     },
+    city: {
```

Comment:
Is this case the prop `city` is created, but isn't sent in the request.

Can you check this?